src/components/Form.js

```diff
@@ -281,8 +281,10 @@ class Form extends React.Component {
                 value: this.state.inputValues[inputID],
                 errorText: this.state.errors[inputID] || fieldErrorMessage,
                 onBlur: () => {
-                    this.setTouchedInput(inputID);
-                    this.validate(this.state.inputValues);
+                    setTimeout(() => {
```

Comment:
We should add a comment explaining why we are adding this timeout to give context to other engineers that come across this code

Reply:
Added the comment!